prd.md

```diff
@@ -0,0 +1,119 @@
+# Co-Hosting Feature PRD
+
+## Overview
+
+Add support for multiple hosts (co-hosts) to events in Shera, where all hosts have equal permissions and capabilities.
+
+## Feature Details
+
+### Core Functionality
+
+- All hosts have equal permissions and capabilities
```

Comment:
I think you'd want the possibility of having a destinction between "main" host and "co-host". 

I.e. an arrangement could be hosted by someone as main actor, but with the "help" from others. You probably don't want the "guest" hosts to have the ability to take over as main host without the main host requesting this

Reply:
I did consider this and concluded that I wouldn't run multiple levels of privilege. It comes with a significant increase in complexity, and it does not really seem necessary. The worst thing that can happen here is that someone you already gave access to cohost the event gets control of a Shera event, which really isn't that big a deal in the first place. 

A big part of my development philosophy for this project is to not preemptively build complexity. Keep things light and breezy and it's easy to expand stuff like this later if needed.